getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Getting started with VPP networking
+description: Install Calico with the VPP dataplane on a Kubernetes cluster
+canonical_url: '/getting-started/kubernetes/vpp/getting-started'
+---
+
+
+### Big picture
+
+Install {{site.prodname}} and enable the tech preview of the new VPP dataplane.
```

Comment:
Install {{site.prodname}} and enable the tech preview of the VPP dataplane.

(I suggest removing "new" because a tech previews can take awhile before GA).